### PR TITLE
feat(#128): Fix requirements and paths

### LINKED ISSUE #128: [Bug]: Traffic light training does not work in container
### Current Behavior

```
$ b5 shell
$ cd /workspace/code/perception/src/traffic_light_detection
$ python src/traffic_light_detection/traffic_light_training.py
```

should train the model

### Expected Behavior

Command will work and returns a trained model.

### How to reproduce the issue

Run the commands as described above.

### PR DESCRIPTION
**Pull request by timdreier**
# Description

It was not possible to train a new model because some dependencies where missing and the required package was not added to PATH.

Did some research, what's the best way to add some custom packages to PATH.

Fixes #128 

## Time invested

Tim Dreier: 45 min

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## Does this PR introduce a breaking change?

No.

## Most important changes

```
$ b5 shell
$ cd /workspace/code/perception/src/traffic_light_detection
$ python src/traffic_light_detection/traffic_light_training.py
```

works now.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (might be obsolete with CI later on)
- [x] New and existing unit tests pass locally with my changes (might be obsolete with CI later on)

### CODE REVIEW

File: code/perception/src/traffic_light_detection/src/traffic_light_detection/traffic_light_training.py [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
 from torchvision.datasets import ImageFolder
-from data_generation.transforms import Normalize, ResizeAndPadToSquare, \
-    load_image
-from data_generation.weights_organizer import WeightsOrganizer
-from traffic_light_detection.classification_model import ClassificationModel
 from torchvision.transforms import ToTensor
-from traffic_light_config import TrafficLightConfig
+import sys
+import os
+sys.path.append(os.path.abspath(sys.path[0] + '/..'))
```
Comment (nylser) [inline code/perception/src/traffic_light_detection/src/traffic_light_detection/traffic_light_training.py:12]: This should not be required when you are installing the dependencies correctly via requirements.txt. Did you encounter an error in PyCharm or during execution in the built container?

**Comment by JoKircher** [comment]
Does this PR need changes? There is a merge conflict with main. Otherwise, I would happily review it when finished.

**Review by JoKircher — CHANGES_REQUESTED** [review]
Nice PR! It seems to solve the problem with requirements, but I do get another error message when running the commands described in #128 locally.

```
Traceback (most recent call last):
  File "src/traffic_light_detection/traffic_light_training.py", line 151, in <module>
    tr = TrafficLightTraining(cfg)
  File "src/traffic_light_detection/traffic_light_training.py", line 55, in __init__
    self.train_dataset = ImageFolder(root=self.cfg.DATASET_PATH + "/train",
  File "/home/carla/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 309, in __init__
    super().__init__(
  File "/home/carla/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 144, in __init__
    classes, class_to_idx = self.find_classes(self.root)
  File "/home/carla/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 218, in find_classes
    return find_classes(directory)
  File "/home/carla/.local/lib/python3.8/site-packages/torchvision/datasets/folder.py", line 40, in find_classes
    classes = sorted(entry.name for entry in os.scandir(directory) if entry.is_dir())
FileNotFoundError: [Errno 2] No such file or directory: '/workspace/code/perception/src/traffic_light_detection/dataset/train'
```
Could be please point out, if this is a mistake on my end?

review time so far: 30 min

**Comment by timdreier** [comment]
@JoKircher the command `dvc pull` is missing in the instructions above. Sorry for that :) Run this and then proceed as already.

**Review by JoKircher — APPROVED** [review]
Nice PR! now works as suggested. Before merging there are some merge conflicts to eliminate first!

**Pull request merged**